deploy/distrib/src/main/resources/wildfly/modules/system/layers/base/sun/jdk/main/module.xml

```diff
@@ -0,0 +1,123 @@
+                <path name="com/sun/rowset"/>
+                <path name="com/sun/rowset/providers"/>
+                <path name="com/sun/security/auth"/>
+                <path name="com/sun/security/auth/login"/>
+                <path name="com/sun/security/auth/module"/>
+                <path name="com/sun/tools/internal/xjc"/>
+                <path name="com/sun/security/jgss"/>
+                <path name="sun/awt"/>
+                <path name="sun/awt/color"/>
+                <path name="sun/awt/datatransfer"/>
+                <path name="sun/awt/dnd"/>
+                <path name="sun/awt/event"/>
+                <path name="sun/awt/geom"/>
+                <path name="sun/awt/im"/>
+                <path name="sun/awt/image"/>
+                <path name="sun/awt/image/codec"/>
+                <path name="sun/awt/motif"/>
+                <path name="sun/awt/resources"/>
+                <path name="sun/awt/shell"/>
+                <path name="sun/awt/util"/>
+                <path name="sun/awt/windows"/>
+                <path name="sun/awt/X11"/>
+                <path name="sun/dc"/>
+                <path name="sun/print"/>
+                <path name="sun/print/resources"/>
+                <path name="sun/security/action"/>
+                <path name="sun/security/pkcs"/>
+                <path name="sun/security/x509"/>
+                <path name="sun/jdbc/odbc"/>
+                <path name="sun/jdbc/odbc/ee"/>
+                <path name="sun/invoke"/>
+                <path name="sun/font"/>
+                <path name="sun/misc"/>
+                <path name="sun/io"/>
+                <path name="sun/nio"/>
+                <path name="sun/nio/ch"/>
+                <path name="sun/nio/cs"/>
+                <path name="sun/nio/cs/ext"/>
+                <path name="sun/rmi/log"/>
+                <path name="sun/rmi/registry"/>
+                <path name="sun/rmi/runtime"/>
+                <path name="sun/rmi/server"/>
+                <path name="sun/rmi/transport"/>
+                <path name="sun/rmi/transport/proxy"/>
+                <path name="sun/rmi/transport/tcp"/>
+                <path name="sun/security"/>
+                <path name="sun/security/util"/>
+                <path name="sun/security/krb5"/>
+                <path name="sun/security/krb5/internal"/>
+                <path name="sun/util"/>
+                <path name="sun/util/calendar"/>
+                <path name="sun/util/locale"/>
+                <path name="sun/util/resources"/>
+                <path name="sun/security/pkcs11"/>
+                <path name="sun/security/provider"/>
+                <path name="sun/text"/>
+                <path name="META-INF/services"/>
+            </paths>
+            <exports>
+                <include-set>
+                    <path name="META-INF/services"/>
+                </include-set>
+            </exports>
+        </system>
+    </dependencies>
+</module>
```

Comment:
it tells wildfly which jdk package to include by default
we modify it because we need some other package than the wildfly default ones